server/src/test/java/org/apache/druid/metadata/IndexerSQLMetadataStorageCoordinatorTest.java

```diff
@@ -3048,21 +3159,23 @@ private List<DataSegment> createAndGetUsedYearSegments(final int startYear, fina
     return segments;
   }
 
-  private ImmutableList<DataSegment> retrieveUnusedSegmentsUsingMultipleIntervalsAndLimit(
+  private ImmutableList<DataSegment> retrieveUnusedSegmentsUsingMultipleIntervalsLimitAndOffset(
```

Comment:
There's no offset anymore; maybe we can simply call this method `retrieveUnusedSegments`?